src/changelog/2023/11/default-editor.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Device Editor enabled by default
+date: 2023-11-23 11:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+With the latest update, devices operating in `Developer Mode` will now have the `Open Editor` feature enabled by default. This enhancement is designed to improve the User Experience.
```

Comment:
Isn't it the other way around? For each device there's a editor button which will enable developer mode implicitly

Reply:
No

Reply:
![Screenshot 2023-11-23 at 11 43 52](https://github.com/FlowFuse/website/assets/73583313/c4ee4139-7f6c-4af3-b61d-b1440d8a940a)
![Screenshot 2023-11-23 at 11 44 03](https://github.com/FlowFuse/website/assets/73583313/e8c985d2-d95c-44c3-93c7-43bf231a4050)